src/pybind/mgr/rbd_support/module.py

```diff
@@ -117,21 +118,57 @@ class Module(MgrModule):
                    "name=task_id,type=CephString,req=false ",
             "desc": "List pending or running asynchronous tasks",
             "perm": "r"
+        },
+        {
+            "cmd": "rbd trash purge schedule add "
+                   "name=level_spec,type=CephString "
+                   "name=interval,type=CephString "
+                   "name=start_time,type=CephString,req=false ",
+            "desc": "Add rbd trash purge schedule",
+            "perm": "w"
+        },
+        {
+            "cmd": "rbd trash purge schedule remove "
+                   "name=level_spec,type=CephString "
+                   "name=interval,type=CephString,req=false "
+                   "name=start_time,type=CephString,req=false ",
+            "desc": "Remove rbd trash purge schedule",
+            "perm": "w"
+        },
+        {
+            "cmd": "rbd trash purge schedule list "
+                   "name=level_spec,type=CephString ",
+            "desc": "List rbd trash purge schedule",
+            "perm": "r"
+        },
+        {
+            "cmd": "rbd trash purge schedule dump",
```

Comment:
Should this match the final implementation from the mirror snap scheduler? e.g. `dump` -> `list` w/ the extra parameters